src/cmd/projectDelete.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"context"
+
+	"time"
+
+	"github.com/zerops-io/zcli/src/constants"
+	"github.com/zerops-io/zcli/src/grpcApiClientFactory"
+
+	"github.com/zerops-io/zcli/src/cliAction/startStopDeleteProject"
+
+	"github.com/zerops-io/zcli/src/i18n"
+	"github.com/zerops-io/zcli/src/utils/httpClient"
+	"github.com/zerops-io/zcli/src/utils/zipClient"
+
+	"github.com/spf13/cobra"
+)
+
+var confirm bool
+
+func deleteProjectCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "delete project [projectName] --confirm",
+		Short:        i18n.CmdDeleteProject,
+		Args:         cobra.MinimumNArgs(1),
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx, cancel := context.WithCancel(context.Background())
+			regSignals(cancel)
+
+			storage, err := createCliStorage()
+			if err != nil {
+				return err
+			}
+
+			region, err := createRegionRetriever()
+			if err != nil {
+				return err
+			}
+
+			reg, err := region.RetrieveFromFile()
+			if err != nil {
+				return err
+			}
+
+			apiClientFactory := grpcApiClientFactory.New(grpcApiClientFactory.Config{
+				CaCertificateUrl: reg.CaCertificateUrl,
+			})
+			apiGrpcClient, closeFunc, err := apiClientFactory.CreateClient(
+				ctx,
+				reg.GrpcApiAddress,
+				getToken(storage),
+			)
+			if err != nil {
+				return err
+			}
+			defer closeFunc()
+
+			client := httpClient.New(httpClient.Config{
+				HttpTimeout: time.Minute * 15,
+			})
+
+			zip := zipClient.New(zipClient.Config{})
+
+			return startStopDeleteProject.New(
+				startStopDeleteProject.Config{},
+				client,
+				zip,
+				apiGrpcClient,
+			).Run(ctx, startStopDeleteProject.RunConfig{
+				ProjectName: args[1],
+			}, getActionType())
+		},
+	}
+
+	params.RegisterString(cmd, "projectName", "", i18n.ProjectName)
+
+	cmd.Flags().BoolVarP(&confirm, "confirm", "c", false, "confirm delete project")
+
+	return cmd
+}
+
+func getActionType() string {
```

Comment:
I would maybe use `bool` here already (like `confirmed`) and not deduce if it's confirmed by string prefix. Do you need the `YDelete` somewhere else also or is it just for this usecase?